### PR TITLE
USUM Update

### PR DESCRIPTION
**Pull request by KrisOnline**

### CODE REVIEW

**Comment by KrisOnline** [comment]
Thanks to @HoeenCoder and @Marty-D for helping

File: data/formats-data.js [hunk 1]
```diff
@@ -6480,6 +6510,22 @@ exports.BattleFormatsData = {
 		eventOnly: true,
 		tier: "Uber",
 	},
+	poipole: {
+		tier: "NFE",
```
Comment (TheFenderStory) [inline data/formats-data.js]: Does poipole need to be event only? I don't believe it is supposed to be useable in Little Cup.

Comment (KrisOnline) [reply data/formats-data.js]: I'm waiting on adding the eventOnly because i dont know the event yet; itll be updated as soon as it's all confirmed

Comment (CTCC1) [reply data/formats-data.js]: It is a gift obtained in game with level 40, so it is not useable in Little Cup. Most of those walkthrough videos on youtube can confirm this.

Comment (TheFenderStory) [reply data/formats-data.js]: Yep we found the event set. All good.

**Comment by urkerab** [comment]
Where's rockruffdusk's template which says its ability is Own Tempo?
Why is rockruffdisk event only if lycanrocdusk has an egg group?
If you have both multiple kyurems and necrosma formes, should you get all the errors at once?

**Comment by KrisOnline** [comment]
>Where's rockruffdusk's template which says its ability is Own Tempo?
is rockruffdisk event only if lycanrocdusk has an egg group?

I was told by Zarel to handle Rockruff-Dusk like this, it doesn't work but USUM comes out/is out already so it'll be fixed later. You can at the very least use both.

> If you have both multiple kyurems and necrosma formes, should you get all the errors at once?

no....

**Comment by Zarel** [comment]
> If you have both multiple kyurems and necrosma formes, should you get all the errors at once?

That would be marginally useful: I'd do it if it didn't complicate the code, but it does, so this is fine.

**Pull request merged**